### PR TITLE
Add video to home Hero

### PR DESCRIPTION
**Pull request by taniaprocha**
- update oncostats image

### CODE REVIEW

File: src/components/Background/index.css [hunk 1]
```diff
@@ -17,41 +19,77 @@
...
   }
 }
 
-.color-lavender::after {
+.color-lavender {
   background-color: color($lavender alpha(84%));
 }
 
-.color-light-blue::after {
+.color-light-blue {
```
Comment (filipebfernandes) [inline src/components/Background/index.css]: Also remove the ::after for the color-magenta

**Comment by jfranciscosousa** [comment]
Is the video size final?

File: src/components/Background/index.css (hunk 1 above)
Comment (taniaprocha) [reply src/components/Background/index.css]: 👍

**Comment by taniaprocha** [comment]
No @jfranciscosousa, this isn't the final movie version, Inês has to solve loop issues and make a new video.

**Pull request merged**